### PR TITLE
Fix docs about table prefixes

### LINKED ISSUE #8354: Improve table prefix documentation
The examples in [the ActiveRecord guide](http://www.yiiframework.com/doc-2.0/guide-db-active-record.html#declaring-active-record-classes) and [the ActiveRecord class documentation](http://www.yiiframework.com/doc-2.0/yii-db-activerecord.html) don't use `{{% }}`.
[The documentation for tableName](http://www.yiiframework.com/doc-2.0/yii-db-activerecord.html#tableName%28%29-detail) doesn't mention it. In fact it implies that the default implementation adds the table prefix directly, when actually it just adds the `{{% }}`.
[The documentation for \yii\db\Query](http://www.yiiframework.com/doc-2.0/yii-db-query.html#from%28%29-detail) and the example there also completely omits it.
Only [the documentation for \yii\db\Connection::tablePrefix](http://www.yiiframework.com/doc-2.0/yii-db-connection.html#$tablePrefix-detail) mentions it, but it's not really clear where you can use `{{% }}`, since it just vaguely refers to "a table name". I knew that I can use `{{% }}` in `ActiveRecord::tableName()`, but only because gii has an option to generate it like that, and I was not aware that I could use it in `Query::from()`.
Wouldn't it be better to use `{{% }}` in all these examples?

### PR DESCRIPTION
**Pull request by dynasource**
fixes #8354 by adding docs about prefixes to the mentioned areas

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: docs/guide/db-active-record.md [hunk 1]
```diff
@@ -50,9 +50,18 @@ However, most content described here are also applicable to Active Record for No
 
 ## Declaring Active Record Classes <span id="declaring-ar-classes"></span>
 
-To get started, declare an Active Record class by extending [[yii\db\ActiveRecord]]. Because each Active Record
-class is associated with a database table, in this class you should override the [[yii\db\ActiveRecord::tableName()|tableName()]]
-method to specify which table the class is associated with.
+To get started, declare an Active Record class by extending [[yii\db\ActiveRecord]]. 
+
+### Setting a table name
+By default each Active Record class is associated with it's database table,
```
Comment (SilverFire) [inline docs/guide/db-active-record.md]: `it's` -> `its`

File: docs/guide/db-active-record.md [hunk 2]
```diff
@@ -50,9 +50,18 @@ However, most content described here are also applicable to Active Record for No
...
 ## Declaring Active Record Classes <span id="declaring-ar-classes"></span>
 
-To get started, declare an Active Record class by extending [[yii\db\ActiveRecord]]. Because each Active Record
-class is associated with a database table, in this class you should override the [[yii\db\ActiveRecord::tableName()|tableName()]]
-method to specify which table the class is associated with.
+To get started, declare an Active Record class by extending [[yii\db\ActiveRecord]]. 
+
+### Setting a table name
+By default each Active Record class is associated with it's database table, 
+[[yii\db\ActiveRecord::tableName()|tableName()]] returns the class name as the table name by calling [[Inflector::camel2id()]].
```
Comment (SilverFire) [inline docs/guide/db-active-record.md]: `[[yii\db\ActiveRecord::tableName()|tableName()]] returns` -> `[[yii\db\ActiveRecord::tableName()|tableName()]] method returns`

Comment (SilverFire) [inline docs/guide/db-active-record.md]: Namespace for `[[Inflector]]` is missing

**Review by SilverFire — CHANGES_REQUESTED** [review]

**Review by SilverFire — APPROVED** [review]

File: docs/guide/db-query-builder.md [hunk 3]
```diff
@@ -153,6 +153,12 @@ $subQuery = (new Query())->select('id')->from('user')->where('status=1');
 $query->from(['u' => $subQuery]);
 ```
 
+#### Prefixes
+Also a default [[yii\db\Connection::$tablePrefix|tablePrefix]] can be applied. For example if
+ [[yii\db\Connection::$tablePrefix|tablePrefix]] is `tbl_`, `Customer` becomes `tbl_customer` and `OrderItem` becomes `tbl_order_item`.
+ 
+If a table name is given as `{{%TableName}}`, then the percentage character `%` will be replaced with this property value. 
+For example, `{{%post}}` becomes `{{tbl_post}}`.
```
Comment (cebe) [inline docs/guide/db-query-builder.md]: This is already covered in `db-dao.md`. http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names

**Comment by cebe** [comment]
I have added a commit with some improvements. Also table prefix in query guide seems duplicate, it should link to http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names instead.

**Review by cebe — APPROVED** [review]

**Pull request merged**